src/test/resources/application-test.yml

```diff
@@ -14,10 +14,12 @@ spring:
   flyway:
     locations: classpath:db/migration/all,classpath:db/migration/integration
   jpa:
+    show-sql: true
```

Comment:
do you mean to leave this enabled? I think it's helpful but wonder if it'll slow down tests in circle?

Reply:
Whoops!